lib/forms.js

```diff
@@ -44,7 +44,7 @@ var forms = {
         });
       },
       mongooseConnection: async.apply(setupConnections.getMongooseConnection, uri, forms.sharedConnections.mongooseConnection),
-      mongoConnection: async.apply(setupConnections.getMongoConnection, uri, forms.sharedConnections.databaseConnection)
+      mongoConnection: async.apply(setupConnections.getMongoConnection, uri, forms.sharedConnections.mongodbConnection)
```

Comment:
should match the value here: https://github.com/feedhenry/fh-forms/blob/master/lib/forms.js#L104